employees.json

```diff
@@ -0,0 +1,164 @@
+{
```

Comment:
What's this copy doing at this point? Can we delete it?

Reply:
so this lives here so that way when people do --discovery, this is what it uses. since we have the --discover feature in the readme, i figured we'd want this

https://github.com/singer-io/getting-started/blob/master/docs/DISCOVERY_MODE.md

Reply:
@alisaraa looking at the Python code, it looks like the more complete copy at `tap_paycor/schemas/employees.json` is what gets returned in `--discovery` mode